.circleci/config.yml

```diff
@@ -1510,55 +1596,72 @@ workflows:
       #
       # Unit tests with pytest on Linux
       #
-      - tox-base:
+      - unit-tests-new:
           requires:
-            - "unit-tests"
+            - unit-tests
```

Comment:
btw unit-tests is just an empty job to organize the tests visually, so you can rename it to `unit-test-root` and call `unit-tests-new` just `unit-tests` (also now that we have fewer tests, we could theoretically get rid of it, tho it helps to see all the unit test fan out in the UI)

Reply:
i removed the root job! it still looks readable in the CircleCI workflow view to me